holidays/countries/syrian_arab_republic.py

```
@@ -0,0 +1,142 @@
+# A fast, efficient Python library for generating country, province, and state
+# specific sets of holidays on the fly.
+#
+# Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+# Website: https://github.com/vacanza/holidays
+# License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import (
+    JAN,
```

Ruff (0.8.2)

`holidays.calendars.gregorian.JAN` imported but unused

Remove unused import

(F401)